app/views/pages/privacy.html.slim

```diff
@@ -0,0 +1,87 @@
+h1 Privacy Policy
+
+em Last updated: May 25th, 2018
+
+h2 Introduction
+p We take the protection of our users (“User/you/your”) personal data very seriously and strictly comply with applicable data protection laws and regulations. In our privacy policy below (“Privacy Policy”) we provide you with an overview of what data we collect for what purpose and how we ensure the protection of the data. 
+
+p The controller is <strong>Travis Foundation gUG (haftungsbeschränkt)</strong>, Rigaer Straße 8, 10247 Berlin/Germany, registered at the local court (Amtsgericht) of Charlottenburg under HRB 158047 B), represented by the managing director Anika Lindtner (“we/us/our”).
+
+h2 Information gathering and usage
+
+p When registering for the <a href="https://teams.railsgirlssummerofcode.org">Rails Girls Summer of Code Teams Application</a> (referred to as RGSoC from now), we collect personally identifiable information such as your GitHub username and email address.
```

Comment:
Why is this a link?

Reply:
this was a suggestion by Maria, am happy to remove the link if you think it's redundant. :)